docs/source/release-notes/2020-2-3-Asteria.md

```diff
@@ -0,0 +1,93 @@
+* API: `Entities.pasteEntities` now allows you to paste to any host type, defaulting to domain if not specified. (#497)
+* Graphics: Particles now pause when you look away instead of dying and restarting. (#498)
+* API: `Entities.importEntities` now allows you to import entities to your clipboard even if you have no create rights on the current domain. (#504)
+* Graphics: The maximum texture size has been upgraded from 2048 -> 8192. (#505)
+* App: The Explore app has been updated from its main repo. (#536)
+
+#### Technicals
+
+* Building: Android building has been fixed by updating the URLs for packages. (#466)
+* Docs: The general build documentation was updated to better reflect build variables. (#487)
+* Docs: The OSX build documentation was updated to better define instructions. (#499, #533)
+
+## Domain Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+#### General
+
+* Stability: A potential crash fix was applied to the entity server. (#435)
+* UI: The descriptions for the Sandbox's get/set private user data fields was improved. (#486)
+
+## Metaverse Server
+
+While the [metaverse server](https://github.com/kasenvr/project-apollo) is still in development, much progress has been made since starting. The project has over **150 commits** at the time of publishing these release notes, with over 20 commits in the last two weeks (July 1st - July 14th, 2020). Building upon the foundation that Zontreck had placed, Misterblue has created a working prototype of the metaverse. After adequate testing and completion of the system is complete, we will begin the transition to our very own federated metaverse system!
+
+The [metaverse server interface](https://github.com/kasenvr/project-apollo-dashboard) has also gotten underway this month. It will serve as the UI foundation for admins and users alike to access and administrate their identities on the metaverse. It has had **18 commits** in the last two weeks (July 1st - July 14th, 2020).
+
+## Extra Applications
+
+#### Explore App (Interim Version)
+
+The numbers at the end of each item are the PR numbers in the Decentralized GoTo [repo](https://github.com/kasenvr/Decentralized_GoTo_Experimental).
+
+* Update the Explore app's sorting order to be higher in the menu. (#5)
+
+#### Community Apps
+
+The numbers at the end of each item are the PR numbers in the Community Apps [repo](https://github.com/kasenvr/community-apps). 
+
+* The nametags app by CtrlAltDavid has been added to the community apps repo. This app allows you to have dynamically generated nametags. (#20)
+
+#### Inventory App
+
+This is a new app that will be introduced to the community apps repo upon merging into the main project-athena codebase (#521). It will serve as the initial implementation of an inventory system into Vircadia, later it will incorporate the functions of the "More" app, that app will be deprecated at such a time.
```

Comment:
```suggestion
This is a new app that will be introduced to the community apps repo upon merging into the main project-athena codebase (#521). It will serve as the initial implementation of an inventory system into Vircadia, later it will incorporate the functions of the "More" app; that app will be deprecated at such a time.
```